packages/now-static-build/src/index.ts

```diff
@@ -120,10 +120,10 @@ function getCommand(
     return null;
   }
 
-  const propName = name === 'build' ? 'buildCommand' : 'devCommand';
+  const propName = `${name}Command`;
```

Comment:
I think you can avoid the any below if you do 
```suggestion
  const propName = `${name}Command` as typeof keyof config;
```

Reply:
That worked for the "config" object, but not for the "framework".